src/test/java/seedu/address/logic/commands/GetCommandTest.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for GetCommand.
+ */
+public class GetCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_getPersonAtIndex_success() {
+        String expectedMessage = String.format(GetCommand.MESSAGE_GET_PERSON_SUCCESS, 1);
+        GetCommand command = new GetCommand(INDEX_FIRST_PERSON);
+        List<Person> lastShownList = model.getFilteredPersonList();
+        Person personToGet = lastShownList.get(INDEX_FIRST_PERSON.getZeroBased());
+        Predicate<Person> predicate = x -> personToGet.equals(x);
+        model.updateFilteredPersonList(predicate);
+        assertCommandSuccess(command, model, expectedMessage, model);
```

Comment:
In line 40, should the second "model" be "expectedModel" (and the "model"s in the previous lines should be replaced by "expectedModel")? I believe that in assertCommandSuccess, the first "model" is modified by the command and the second "model" is modified by the predicate.